variants/esp32s3-devkit-lipo/pins_arduino.h

```diff
@@ -0,0 +1,78 @@
+#ifndef Pins_Arduino_h
+#define Pins_Arduino_h
+
+#include <stdint.h>
+#include "soc/soc_caps.h"
+
+#define USB_VID 0x303a
+#define USB_PID 0x1001
+
+static const uint8_t LED_BUILTIN = 38;
+#define BUILTIN_LED  LED_BUILTIN // backward compatibility
+#define LED_BUILTIN  LED_BUILTIN // allow testing #ifdef LED_BUILTIN
+
+static const uint8_t BUT_BUILTIN = 0;
+#define BUILTIN_BUT  BUT_BUILTIN // backward compatibility
+#define BUT_BUILTIN  BUT_BUILTIN // allow testing #ifdef BUT_BUILTIN
+
+static const uint8_t TX = 43;
+static const uint8_t RX = 44;
+
+// available at pUEXT +
+	// UART
+	static const uint8_t TX1 = 17;
+	static const uint8_t RX1 = 18;
+
+	// I2C
+	static const uint8_t SDA = 48;
+	static const uint8_t SCL = 47;
+
+	// SPI
+	static const uint8_t SS    = 10;
+	static const uint8_t MOSI  = 11;
+	static const uint8_t MISO  = 13;
+	static const uint8_t SCK   = 12;
```

Comment:
Formatting.
```suggestion
// UART
static const uint8_t TX1 = 17;
static const uint8_t RX1 = 18;

// I2C
static const uint8_t SDA = 48;
static const uint8_t SCL = 47;

// SPI
static const uint8_t SS    = 10;
static const uint8_t MOSI  = 11;
static const uint8_t MISO  = 13;
static const uint8_t SCK   = 12;
```

Reply:
Thank you for the suggestions, I made the changes.
Let me know if there are other issues.